index.html

```diff
@@ -23,9 +23,13 @@
         
         <script type="text/javascript" src="javascript/custom/ng-currency.js"></script>
         
-        <script type="text/javascript" src="javascript/retirement_calculator_module.js"></script>
-        <script type="text/javascript" src="javascript/create_retirement_graph_module.js"></script>
         <script type="text/javascript" src="javascript/app.js"></script>
+        <script type="text/javascript" src="javascript/date_service.js"></script>
+        <script type="text/javascript" src="javascript/interest_service.js"></script>
+        <script type="text/javascript" src="javascript/geometry_service.js"></script>
+        <script type="text/javascript" src="javascript/retirement_calculator_service.js"></script>
+        <script type="text/javascript" src="javascript/create_retirement_graph_service.js"></script>
+        

```

Comment:
For now, yes we'll just add another script. But in the long run you're exactly right. We'll build a grunt task that concatenates all our js into one file (and probably minify it as well). That way we'll only have to load one javascript file and one css file, which will improve load time and has a number of other benefits as well.